formatting.md

```diff
@@ -20,15 +20,24 @@ developers may find it more productive to:
 The alternative is often tiring formatting debates during code reviews, where
 time may be better spent on code behavior rather than code style.
 
-### Automatically formatting code in IntelliJ
+### Automatically formatting code in Android Studio, and IntelliJ
 
-Automatic formatting of code is supported in IntelliJ if you have the
-`Dart` plugin (see [IntelliJ setup](/ide-setup/)).
+Automatic formatting of code is supported in Android Studio, and IntelliJ if you
+have the `Dart` plugin (see [Editor setup](/get-started/setup-editor/)).
 
 To automatically format the code in the current source code window, right-click
 in the code window and select `Reformat with Dart style`. You can add a keyboard
 shortcut to this in Keymap section of IntelliJ Preferences.
 
+### Automatically formatting code in VS Code
+
+Automatic formatting of code is supported in Visual Studio Code if you
```

Comment:
for future: try phrasing like "Install the Dart Code plugin to automatically format your Dart code."

Reply:
Done